src/main/scala/ch/epfl/scala/sbtmissinglink/MissingLinkPlugin.scala

```diff
@@ -31,16 +32,25 @@ object MissingLinkPlugin extends AutoPlugin {
   override def trigger: PluginTrigger = allRequirements
 
   val configSettings: Seq[Setting[_]] = Def.settings(
-    missinglinkCheck := {
+    missinglinkExcludeDependencies := Nil,
+    missinglinkConflicts := {
       val log = streams.value.log
-
-      val cp = Attributed.data(fullClasspath.value)
+      val cp = fullClasspath.value
```

Comment:
This change does not seem necessary, and comes with a lot of downstream changes to manipulate (and create) `Attribute[File]`s instead of plain `File`s. Consider reverting it.

Reply:
this is necessary to exclude dependencies